<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookViewWindow</class>
 <widget class="QWidget" name="BookViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book View - Spire</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/icons/book_blue.png</normaloff>:/icons/book_blue.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="m_technicalsLayout" stretch="1,0,2,0,2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QGridLayout" name="m_highLowLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lowLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Low:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_highLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>High:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_highValue">
         <property name="text">
          <string>99.99</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_lowValue">
         <property name="text">
          <string>99.99</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="m_highQuantitySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="m_volumeLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_volumeValue">
         <property name="text">
          <string>99999999</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_volumeLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Vol:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_defaultQuantityLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Def:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_defaultQuantityValue">
         <property name="text">
          <string>999x999</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="m_quantityOpenSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="m_openLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_openValue">
         <property name="text">
          <string>99.99</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_openLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Open:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_closeLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Close:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Spire::UI::ValueLabel" name="m_closeValue">
         <property name="text">
          <string>99.99</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_panelLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Spire::BookViewPanel" name="m_bidPanel" native="true"/>
     </item>
     <item>
      <widget class="Spire::BookViewPanel" name="m_askPanel" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::BookViewPanel</class>
   <extends>QWidget</extends>
   <header>Spire/BookView/BookViewPanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spire::UI::ValueLabel</class>
   <extends>QLabel</extends>
   <header>Spire/UI/ValueLabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
